person.rb

```diff
@@ -10,6 +10,7 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
   def correct_name
```

Comment:
Same here, kindly make sure that there's a method for adding rentals where the `person` itself is sent as a parameter to create the new rental achieving the 'has-many' association.

- The method is similar to the one in the example above except now `self` should be the argument for the `person` parameter in the Rental instance.